src/Utility.js

```diff
@@ -34,3 +34,15 @@ export function addComment(articleId, comment) {
   return apiLink.post(`/articles/${articleId}/comments`, comment);
   
 }
+
```

Comment:
With this file it might be more appropriate to have a utillity folder with an api.js file inside as the functions in this file have a specific purpose.